src/terraform/examples/vfxt/proxy/main.tf

```diff
@@ -86,6 +88,8 @@ module "network" {
 
   open_external_ports   = local.open_external_ports
   open_external_sources = local.open_external_sources
+  peer_vnet_rg          = local.peer_vnet_rg
```

Comment:
for this to work, the module "render_network_secure" needs the same updates we made to "render_network"